0000-report-inboxes.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: report-inboxes
+- Start Date: 2024-02-20
+- RFC PR: [LemmyNet/rfcs#0000](https://github.com/LemmyNet/rfcs/pull/0000)
+- Lemmy Issue: [LemmyNet/lemmy#0000](https://github.com/LemmyNet/lemmy/issues/0000)
+
+# Summary
+
+Rather than combining all reports into a single report inbox, we should allow users to select whether they are reporting to mods or admins, and we should split reports into different inboxes based on that selection.
+
+# Motivation
+
+The current approach has some shortcomings:
+
+* Users are not currently able to bypass mods and report directly to admins - this may allow mods to conceal instance rule breaking in specific communities
+* Admins are not aware of community rules, so they may wish to take no action for most community rule breaking reports. However, if an admin resolves such a report, the relevant community mods most likely never see it.
+* Different instances may have different rules, but somebody resolving a report on one instance will resolve it for other instances as well, thus potentially resulting in missed reports.
+* Mods might take local action on a report and mark it as resolved even in cases where a user should be banned from the entire instance. In this case, admins are very unlikely to see the report.
+
+# Guide-level explanation
+
+### When creating reports, users will be able to select if it's a mod report, or an admin report (or both)
+
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/9a21b527-6c88-4024-b287-3371d77688f4)
```

Comment:
I think it makes sense to be able to send reports to both, I would keep it as checkboxes

Reply:
If users can ignore the choice between admin or mod, they will simply use the default. That would make the whole RFC moot. So I think there should only be radio buttons for either admin or mod.

Reply:
Having both as an option definitely seems useful.
For example, if someone posts nsfw content and forgets or intentionally marks it as not nsfw, this is something that should be addressed as soon as possible, where having the community mods informed in case admins are not immediately available increases the amount of people that might be able to deal with the issue quickly.

This of course also applies to various other problematic content, but even for the spam example mentioned for lemmy.ml.
Having spam reported to both community mods and site admins can allow community mods to take the content down even before site admins will be to perform a site ban if they're not immediately available.